tests/security/apparmor/aa_complain.pm

```diff
@@ -22,18 +22,22 @@
 # - Put nscd back in enforce mode
 # - Cleanup temporary directories
 # Maintainer: llzhao <llzhao@suse.com>
-# Tags: poo#36880, tc#1621142
+# Tags: poo#36880, tc#1621142, poo#81730, tc#1767574
 
 use strict;
 use warnings;
-use base "consoletest";
+use base "apparmortest";
```

Comment:
Just wondering the replacement is all the apparmor api are moved to **apparmortest** /lib already and will not use **consoletest** anymore in the future?

Reply:
> Just wondering the replacement is all the apparmor api are moved to **apparmortest** /lib already and will not use **consoletest** anymore in the future?

In package **apparmortest** it `use base 'consoletest';`

Reply:
Got it.